### DISCUSSION TITLE
Why two parallel discussions for Phalcon?

### DISCUSSION THREAD

**Discussion by AliShareei** [phalcon/phalcon#98] [General]
Hi guys,

Due to the transfer of the forum section to the Github. 
I think the idea of creating two discussions for `phalcon/cphalcon` and `phalcon/phalcon` is not good. 
They are both the same, especially `v5`, which is supposed to be like `v4`. 

Since the future of Phalcon is in `phalcon/phalcon`, I suggest moving both discussions here.

**Comment by Jeckerson** [comment]
For questions about Phalcon v3/v4 you can visit here (https://github.com/phalcon/cphalcon/discussions) and for Phalcon v5 here (https://github.com/phalcon/phalcon/discussions).

**Reply by Jeckerson** [reply]
> They are both the same, especially v5, which is supposed to be like v4.

They are the same for now, after v5 released, it will have different vector. Also different aspect of questions.

**Reply by AliShareei** [reply]
@Jeckerson, I think both should be merged into `phalcon/phalcon` and create a new category (or 2 categories) for `v3` and `v4` on `phalcon/phalcon`. In my opinion, prevent the scattering of useful information and content on two different sections.
